### PR TITLE
Game Create page

### PR DESCRIPTION
**Pull request by ghost**
Guys, I added to the game create page, and this pull contains cool, responsive CSS, an email field and two buttons for a player to choose black or white. Still working on connecting these to the correct routes / show pages.

### CODE REVIEW

File: app/assets/javascripts/game_moves.js [hunk 1]
```diff
@@ -1,9 +1,18 @@
+$(function () {
+  $('#board td').on( 'click', function (data) {
+      $('.cell-highlight').removeClass('cell-highlight');
+      $(this).toggleClass('cell-highlight');
+      console.log(data);
+  });
+});
+
```
Comment (Zjohn64) [inline app/assets/javascripts/game_moves.js:8]: This should be able to slip right in to the new PR i just created

File: app/controllers/games_controller.rb [hunk 2]
```diff
@@ -19,8 +19,8 @@ def create
 	end
 
 	def update
-		render status: 400, json: {
-			text: 'Invalid Move'
+		render status: 200, json: {
+			text: 'Valid Move'
 		}
```
Comment (Zjohn64) [inline app/controllers/games_controller.rb]: This will be a conflict but nothing too difficult to fix

File: app/views/games/show.html.erb [hunk 3]
```diff
@@ -5,24 +5,26 @@
...
+          <% end %>
       </table>
     </div>
-  </div>
-</div>
+    
+    
 
+  </div>
+</div>
```
Comment (Zjohn64) [inline app/views/games/show.html.erb]: This is also different to what I came up with so we should collaborate sometime this week to combine what we have

File: app/views/games/new.html.erb [hunk 4]
```diff
@@ -1,6 +1,41 @@
...
+        <br />
+        <%= simple_form_for @game, url: games_path do |f| %>
+        <%= f.submit "Play Black", class: 'btn black btn-primary' %>
+        <% end %>
+      </div>
+    </div>
+  </div>
+  </div>
+
+</div>
```
Comment (Zjohn64) [inline app/views/games/new.html.erb:41]: Nice form! Now we have a choice lol

**Review by Zjohn64 — COMMENTED** [review]
Overall great just need to collaborate like we did last week and get it working

File: app/assets/javascripts/game_moves.js [hunk 5]
```diff
@@ -15,13 +24,13 @@ $(function() {
...
-         window.location.reload();
-       }
-      });
+      // .success(function(data){
+      //    if (data.valid_move? === false) {
+      //    alert("Invalid move. Try again.");
+      //    window.location.reload();
+      //  }
+      // });
     }
```
Comment (ulalu) [inline app/assets/javascripts/game_moves.js]: My games js file is slightly different- I was fiddling with it with the intention of getting a successful move to also update the turn value. As Zach said, it seems necessary to pull things together and make sure we have a coherent js file that's referencing the correct routes.

File: app/views/games/new.html.erb [hunk 6]
```diff
@@ -1,6 +1,41 @@
...
+        <br />
+        <%= f.submit "Play White", class: 'btn white btn-primary' %>
+        <% end %>
+      </div>
+      <div class="black">
+        <%= image_tag ("black_Knight.png") %>
+        <br />
+        <%= simple_form_for @game, url: games_path do |f| %>
+        <%= f.submit "Play Black", class: 'btn black btn-primary' %>
+        <% end %>
```
Comment (ulalu) [inline app/views/games/new.html.erb:35]: Okay, so it looks like the next step here is to hook this into the logic I'm working on for turns. I should have that pr up soon.

File: app/controllers/games_controller.rb (hunk 2 above)
Comment (ulalu) [reply app/controllers/games_controller.rb]: So this is going to be obsolete on my next pr, I think, so don't worry about this too much at present.

**Review by ulalu — APPROVED** [review]
Really good work here, but I think you, @Zjohn64  and I need to pair/mob program soon to resolve a few potential conflicts. I don't know if you want to merge beforehand or after, but if you'd like to merge this and then work on it later this week as a team that'd be fine with me.

**Pull request closed without merging**